test/Microsoft.AspNetCore.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -3,15 +3,14 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
```

Comment:
If anyone objects to this test cleanup in a patch, I'll revert changes to this file.